mne/decoding/time_gen.py

```diff
@@ -359,7 +354,7 @@ def predict(self, epochs, test_times=None):
                                  for t_test in t_train]
                                  for t_train in test_times_['slices']]
         # Store all testing times parameters
-        self.test_times_ = test_times_
+        self.test_times = test_times_
```

Comment:
do what you have to do. But try to keep things simple. If it gets too
complex it's maybe a suggestion to avoid the corner feature and focus of
the core business :)